README.md

```diff
@@ -32,12 +31,19 @@ jobs:
       - uses: actions/checkout@v2
       - uses: w3c/spec-prod@v2
         with:
-          GH_PAGES_BRANCH: gh-pages
+          W3C_ECHIDNA_TOKEN: ${{ secrets.ECHIDNA_TOKEN }}
+          # Replace following with appropriate value. See options.md for details.
+          W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-group/2014JulSep/1234.html
```

Comment:
Need to fix line 17 to reflect this

Reply:
Not following this comment. Can you clarify a bit?

Reply:
Line 17 says "deploy to GitHub pages", but the example is about W3C deployment now